src/auro-popover.js

```diff
@@ -16,17 +16,22 @@ import Popover from "./popover";
  *
  * @attr {String} placement - Expects top/bottom - position for popover in relation to the element.
  * @attr {String} for - Defines an `id` for an element in the DOM to trigger on hover/blur.
+ * @attr {boolean} isPopoverVisible - Boolean for if popover is visible or not.
+ * @attr {boolean} isSticky - If true, when user clicks trigger, the popover will persist its visibility. If false, popover will disappear when mouseout over trigger.
  * @slot - Default unnamed slot for the use of popover content
  * @slot trigger - Slot for entering the trigger element into the scope of the shadow DOM
  */
 class AuroPopover extends LitElement {
   constructor() {
     super();
     this.placement = 'top';
+    this.isPopoverVisible = false;
+    this.isSticky = this.hasAttribute('sticky');
```

Comment:
This is not needed. https://lit-element.polymer-project.org/guide/properties#declare

Delete line.

Reply:
Lines deleted.